airflow/models/abstractoperator.py

```diff
@@ -124,7 +124,7 @@ def task_type(self) -> str:
         raise NotImplementedError()
 
     @property
-    def inherits_from_dummy_operator(self) -> bool:
+    def inherits_from_empty_operator(self) -> bool:
         raise NotImplementedError()
```

Comment:
Is it safe to rename this? I imagine there may be operators out there relying on this…

Reply:
I thought of that and I'm not 100% sure.
There isn't a real use for it from users side
the test I added in https://github.com/apache/airflow/pull/22832/commits/fb87d0c63aadbf139a5bee353f4db290c5cc6794 shows a custom operator that uses it